scripts/compile-error-code-page.js

```diff
@@ -0,0 +1,55 @@
+const { ErrorCodes } = require("../lib/error-code");
+
+const fs = require("fs");
+const path = require("path");
+
+const errorCodesAPIReference = path.join(
+  __dirname,
+  "..",
+  "website",
+  "docs",
+  "api",
+  "namespaces",
+  "ErrorCodes.md"
+);
+
+const errorCodeDocumentationPath = path.join(
+  __dirname,
+  "..",
+  "website",
+  "docs",
+  "error-codes.md"
+);
+
+(async function () {
+  let errorCodeMarkdown = (await fs.promises.readFile(errorCodesAPIReference))
+    .toString()
+    .replace("## Variables\n", "# Error Codes\n")
+    .replace(
+      `id: "ErrorCodes"
+title: "Namespace: ErrorCodes"
+sidebar_label: "ErrorCodes"
+sidebar_position: 0
+custom_edit_url: null`,
+      `title: "Error Codes"
+sidebar_position: 3`
+    )
+    .replace(/• `Const`.*\n/g, "")
+    .replace(/\n\n\n/g, "\n\n")
+    .replace(/\n#### Defined in.*\n\n.*error.*\n/g, "");
```

Comment:
I went with regex replacements for a quick and dirty solution. Works well. Would be interesting to work programmatically with the tsdoc API though, if that's possible.